docs/Developers/testing-against-testnet.md

```diff
@@ -17,6 +17,8 @@ A full reference to deployed contracts can be found in the [deployments.json](ht
 | Test Token (TEST ERC20) | 0xB7b1d3cC52E658922b2aF00c5729001ceA98142C |
 | Connext Handler | 0x979588965099F4DEA3CAd850d67ca3356284591e |
 
+> If you find there is no `AddLiquidityFor` method in write contract section, you might need to manually call it with your constructed calldata: 0xe070da09 (method address) + the amount you want to provide in hex + the token address + the router address. See the `input data` section of this transaction for a example (https://rinkeby.etherscan.io/tx/0x0b2fbbf0034fefc3bda7738fa81894e051e015da37343ff7663e7419f6dd1329).
```

Comment:
The contract includes `AddLiquidityFor`, so you can definitely use it. No need to manually construct the calldata.
Also, UI is available for routers to add liquidity, and server requests as well.

Reply:
Yes, There is `AddLiquidityFor` in the contract. But what I actually want to represent is that there is no `AddLiquidityFor` method in Rinkeby Etherscan's contract section:

<img width="1328" alt="sc" src="https://user-images.githubusercontent.com/32592090/168085778-fa400a4d-c016-4390-9a05-a2f193070dda.png">

For now as the connext router UI is out, this is indeed useless and I am going to remove it in the PR. Thanks for the feedback~

Reply:
Closing due to invalid finding.